### DISCUSSION TITLE
GitHub Enterprise Server support

### DISCUSSION THREAD

**Discussion by rzheng-sxm** [github/vscode-github-actions#185] [General]
If vscode-github-actions supports GHES, what are the settings I need to change?

**Comment by felipesu19** [comment]
It does not right now. https://github.com/github/vscode-github-actions/issues/12

**Comment by carlost** [comment]
Looks like experimental support for GHES was added recently.

@felipesu19 do you want any feedback on this feature or is it still too early to even hear about issues with GHES integration?

**Reply by felipesu19** [reply]
Feedback is always welcome in issues, my team no longer owns this so I have no idea what's going to happen with GHES support :(
